<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="image_layout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 75 italic 13pt &quot;Ubuntu Condensed&quot;;
</string>
           </property>
           <property name="text">
            <string>Original Image</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="segmentation_layout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: 75 italic 12pt &quot;Ubuntu Condensed&quot;;</string>
           </property>
           <property name="text">
            <string>Segmentated Image</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="Binary_image_layout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 75 italic 12pt &quot;Ubuntu Condensed&quot;;</string>
           </property>
           <property name="text">
            <string>Binary Image</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="heatmap_layout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">font: 75 italic 12pt &quot;Ubuntu Condensed&quot;;</string>
           </property>
           <property name="text">
            <string>Heat Map</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Controls</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>threshold Value</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="threshold_value">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="singleStep">
           <number>15</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="thershold_value">
          <property name="text">
           <string>150</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>ùúè_thrshold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="thrshold1">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(92, 53, 102);</string>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>ùúè‚ÇÇ_thrshold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="thrshold2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(92, 53, 102);</string>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>r_thrshold</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="thrshold_r">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(92, 53, 102);</string>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="specialValueText">
           <string/>
          </property>
          <property name="keyboardTracking">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="singleStep">
           <number>20</number>
          </property>
          <property name="value">
           <number>150</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="segmentation_btn">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(32, 74, 135);</string>
          </property>
          <property name="text">
           <string>apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
